can-list.js

```diff
@@ -8,14 +8,13 @@ var canEvent = require('can-event-queue/map/map');
 var ObservationRecorder = require('can-observation-recorder');
 
 var CID = require('can-cid');
-var isPromise = require('can-util/js/is-promise/is-promise');
-var makeArray = require('can-util/js/make-array/make-array');
-var assign = require('can-util/js/assign/assign');
-var types = require('can-types');
-var each = require('can-util/js/each/each');
 var canReflect = require('can-reflect');
+var isPromise = canReflect.isPromise;
+var makeArray = canReflect.toArray;
```

Comment:
It’s uncommon for us to write it like that; I would just use `canReflect` where it’s needed.

Reply:
Fixed, many thanks :+1: